cli/commands/scripts_update_config_v2.go

```diff
@@ -343,6 +343,14 @@ func newScriptsUpdateConfigV2Cmd(ec *cli.ExecutionContext) *cobra.Command {
 			}
 			ec.Spinner.Stop()
 			ec.Logger.Infoln("Updated config to version 2")
+
+			if _, err = os.Stat(filepath.Join(ec.MigrationDir, "metadata.yaml")); err == nil || os.IsExist(err) {
+				err = os.Remove(filepath.Join(ec.MigrationDir, "metadata.yaml"))
+				if err != nil {
+					return errors.Wrap(err, "cannot remove metadata.yaml file")
```

Comment:
Maybe don't  "return" with a failure here, since we have already completed the migration. I think It'll be better If we do a "Warning" log here instead of returning with an error.

Reply:
okay